extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -73,18 +73,18 @@ define('ext.wikia.adEngine.adContext', [
 			context.opts.showAds = false;
 		}
 
-		// SourcePoint integration
-		if (context.opts.sourcePointUrl) {
-			context.opts.sourcePoint = isUrlParamSet('sourcepoint') ||
-				isProperGeo(instantGlobals.wgAdDriverSourcePointCountries);
-		}
-
 		// SourcePoint detection integration
-		if (context.opts.sourcePointDetectionUrl) {
+		if (!isUrlParamSet('noexternals') && context.opts.sourcePointDetectionUrl) {
```

Comment:
We do set a `wgNoExternals` global when in `noexternals=1` mode. No need to parse an URL :)